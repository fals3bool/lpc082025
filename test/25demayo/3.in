3
Cebolla 0.50
Fideos 0.50
Tomate 0.50
2 2
Cebolla 1.00
Tomate 1.00
