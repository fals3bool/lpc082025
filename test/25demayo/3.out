Fideos 1.0
